sonar-ui/src/components/Key.js

```diff
@@ -1,29 +1,75 @@
-import React from 'react'
+import React, { useState } from 'react'
 import {
-  Text
+  Text,
+  Tooltip,
+  Flex,
+  Icon,
+  Box,
+  useClipboard,
+  Badge
 } from '@chakra-ui/core'
 
 // import './Key.css'
 
 export default function Key (props) {
   const { k: key } = props
   const shortkey = formatKey(key)
-  return (
-    <Text {...props} onClick={e => copyToClipboard(key)}>
-      {shortkey}
-    </Text>
-  )
+  const [copyValue, setCopyValue] = useState(key)
+  const { value, onCopy, hasCopied } = useClipboard(copyValue)
+  // var ttColor
+  if (!hasCopied) {
+    return (copyClipboardTooltip())
+  }
+  return (copiedBadge())
```

Comment:
This style is a bit unusual for React components, but also doesn't really matter much

Reply:
what would be better? this?
```suggestion
  return (hasCopied ? copiedBadge() : copyClipboardTooltip() )
```